projects/sys/default.wmf.yaml

```diff
@@ -10,10 +10,124 @@ paths:
     x-modules:
       - path: sys/page_save.js
   /parsoid:
+    x-modules:
+      - path: sys/parsoid_proxy.js
+        options:
+          parsoidPhpHost: '{{options.parsoid.host_php}}'
+  /parsoidjs:
     x-modules:
       - path: sys/parsoid.js
         options:
           parsoidHost: '{{options.parsoid.host}}'
+          bucketName: 'parsoid'
```

Comment:
Instead of copy/pasting the options, it's better to use YAML references. For `/parsoidjs`, add an anchor to options:

```yaml
options: &parsoidopts
```

And then reuse them below for `/parsoidphp` back-filling the differences:

```yaml
options:
  <<: *parsoidopts
  bucketName: parsoidphp
  parsoidHost: '{{options.parsoid_php_host}}'
```

Reply:
Much better :)